src/modules/transaction/components/TransactionDetailRow/TransactionDetailRow.css

```diff
@@ -0,0 +1,167 @@
+@import '../../../../../setup/react/app/mixins.css';
+
+.container {
+  display: flex;
+  width: 100%;
+  position: relative;
+
+  & > div:first-child {
+    padding: 20px 22px 20px 0px;
+    color: var(--color-content-light);
+    font-weight: var(--font-weight-normal);
+    font-size: 14px;
```

Comment:
We have a set of mixins that help us ensure we have a limited predefined set of typographical elements. I'd recommend using them.

Reply:
done